<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigCard</class>
 <widget class="QWidget" name="frmConfigCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tableCard">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameRight">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="btnSearch">
          <property name="text">
           <string>搜索在线从机</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="btnRemove">
          <property name="text">
           <string>移除 》》</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnSelectAll">
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnSelectNone">
          <property name="text">
           <string>反选</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btnConfirm">
          <property name="text">
           <string>确认设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnRestart">
          <property name="text">
           <string>系统重启</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSynID">
          <property name="text">
           <string>ID同步</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnScreenChar">
          <property name="text">
           <string>屏幕字符</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnResolution">
          <property name="text">
           <string>分辨率/色彩</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSystemInit">
          <property name="text">
           <string>系统初始化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
